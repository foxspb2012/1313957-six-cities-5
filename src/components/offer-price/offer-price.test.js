import React from 'react';
import renderer from 'react-test-renderer';
import OfferPrice from './offer-price';

const PRICE = 120;
const Type = {
  OFFER_PAGE_CONTENT_CARD: `place-card`,
  OFFER_CARD: `property`,
};

describe(`Should OfferPrice render correctly`, () => {
  it(`Should OfferPrice render correctly offer-page-content card typeClass`, () => {
    const tree = renderer
      .create(
          <OfferPrice
            typeClass={Type.OFFER_PAGE_CONTENT_CARD}
            price={PRICE}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Should OfferPrice render correctly offer card typeClass`, () => {
    const tree = renderer
      .create(
          <OfferPrice
            typeClass={Type.OFFER_CARD}
            price={PRICE}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
