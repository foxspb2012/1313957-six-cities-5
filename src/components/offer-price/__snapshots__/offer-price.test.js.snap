// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should OfferPrice render correctly Should OfferPrice render correctly offer card typeClass 1`] = `
<div
  className="property__price"
>
  <b
    className="property__price-value"
  >
    €
    120
  </b>
  <span
    className="property__price-text"
  >
     night
  </span>
</div>
`;

exports[`Should OfferPrice render correctly Should OfferPrice render correctly offer-page-content card typeClass 1`] = `
<div
  className="place-card__price"
>
  <b
    className="place-card__price-value"
  >
    €
    120
  </b>
  <span
    className="place-card__price-text"
  >
     night
  </span>
</div>
`;
